pom.xml

```diff
@@ -62,9 +62,10 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.73.3</jenkins.version>
```

Comment:
Is this gratuitous or do we need a new API?

Reply:
Test dep on `workflow-job` requires this.